settings.py

```diff
@@ -932,10 +930,6 @@
 # override for production
 DEFAULT_PROTOCOL = 'http'
 
-####### South Settings #######
-SKIP_SOUTH_TESTS = True
-SOUTH_TESTS_MIGRATE = False
```

Comment:
There is https://pypi.python.org/pypi/django-test-without-migrations/ or the workaround here: https://groups.google.com/forum/#!msg/django-developers/PWPj3etj3-U/kCl6pMsQYYoJ

However it seems moot when we move to 1.8 it's not necessary and we can use `--keepdb` http://stackoverflow.com/questions/25161425/disable-migrations-when-running-unit-tests-in-django-1-7